presentation/src/main/java/me/androidbox/presentation/agenda/screen/AgendaScreenState.kt

```diff
@@ -0,0 +1,11 @@
+package me.androidbox.presentation.agenda.screen
+
+import com.maxkeppeker.sheets.core.models.base.UseCaseState
+import java.time.ZoneId
+import java.time.ZonedDateTime
+
+data class AgendaScreenState(
+    val displayMonth: String = ZonedDateTime.now(ZoneId.systemDefault()).month.toString(),
+    val usersInitials: String = "",
+    val calendarState: UseCaseState = UseCaseState()
```

Comment:
@philipplackner @MKiperszmid This is the calendar state, is it ok to add it here?

Reply:
Personally since it's really UI dependant, I just leave it in the Composable

Since the only use for the state is to "show"

So what I do is:

Create the state on the composable. Ex:
val calendarState = rememberCalendarState() // Or whatever it's called

And then on the button click, I just:

calendarState.show()

No need for a state for it in this case

But that's my opinionated answer
@philipplackner might have a different answer

Reply:
@MKiperszmid Lets wait for @philipplackner opinion on this

Reply:
Yeah any type of state you get via `rememberX` belongs in the UI, since getting that state instance is a composable function

Reply:
I will keep the calendar state in the screen.